pom.xml

```diff
@@ -78,10 +78,10 @@
         <slf4j.version>1.7.30</slf4j.version>
         <httpcomponents.httpclient.version>4.5.13</httpcomponents.httpclient.version>
         <httpcomponents.httpcore.version>4.4.14</httpcomponents.httpcore.version>
-        <hadoop2.version>2.7.7</hadoop2.version>
+        <hadoop2.version>2.8.5</hadoop2.version>
```

Comment:
Just curious if we shouldn't have the same hadoop version as Tinkerpop have (2.7.7)?
https://github.com/apache/tinkerpop/blob/ff50136e00e3a9649c9c9dfa743b58e4f7b74a71/pom.xml#L155

Reply:
I don't should be problem in the case of hadoop, see version policy https://hadoop.apache.org/versioning.html.

Reply:
Make sense